python/sqlcommenter-python/opentelemetry/sqlcommenter/version.py

```diff
@@ -1,15 +1,15 @@
-#!/usr/bin/python
```

Comment:
any reason to remove these lines ? I thought it is to resolve multiple python installations ?

Reply:
Added